cmd/proxy/config/apiConfig/v1_0.toml

```diff
@@ -97,6 +97,7 @@ Routes = [
 [APIPackages.proof]
 Routes = [
     { Name = "/root-hash/:roothash/address/:address", Secured = false, Open = false, RateLimit = 0 },
+    { Name = "/root-hash/:roothash/address/:address/key/:key", Secured = false, Open = false, RateLimit = 0 },
```

Comment:
so by default you want these endpoints to be closed, right?

Reply:
we should have them opened for internal testnets but closed for the mainnet/testnet/devnet versions

Reply:
Changed to open. They will be closed on mainnet/testnet/devnet.